### ISSUE TITLE
Tasks dont show full output when terminal is small

### ISSUE THREAD

**Issue by kevindiclemente** [Microsoft/vscode#66758]
Issue Type: <b>Bug</b>

Tasks don't display the full output from a shell command when the terminal is small, it will also glitch scroll down on task start. Happy to provide a repro, kedicl@microsoft.com

VS Code version: Code 1.30.2 (61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8, 2019-01-07T22:54:13.295Z)
OS version: Windows_NT x64 10.0.17763

**Comment by fedorg** [comment]
![image](https://user-images.githubusercontent.com/4352852/51439539-210ec400-1ccd-11e9-9478-77f5b0ad5373.png)

**Comment by alexr00** [comment]
This is likely a dupe of #57803. Since you're on a recent enough Windows build, can you try using the setting `"terminal.integrated.windowsEnableConpty": true`? It has it's own problems, but it will confirm that #57803 is the cause.

**Comment by alexr00** [comment]
I can reproduce the scroll issue.

**Comment by alexr00** [comment]
I'm seeing the same scrolling behavior in the integrated terminal when using it without tasks. However, I'm not on a sufficiently new enough build to try a stable version of conpty (see #66158, the build number changed). I'll test on the newer Windows Insider build and see if it's still happening there.

**Comment by alexr00** [comment]
In that case, duplicate of #66574.

**Comment by kevindiclemente** [comment]
@alexr00 I was able to unblock myself here by removing a nested bat script:
before: i had foo.bat call bar.bat
now: i invoke bar.bat directly i don't experiance any weird issues
